com.unity.mobile.notifications/Runtime/Android/Plugins/com/unity/androidnotifications/UnityNotificationManager.java

```diff
@@ -931,3 +972,23 @@ private static synchronized Notification removeScheduledNotification(Integer id)
         return mScheduledNotifications.remove(id);
     }
 }
+
+// Provide a wrapper for NotificationChannel.
+// Create this wrapper for all Android versions as NotificationChannel is only available for Android O or above.
+class NotificationChannelWrapper {
```

Comment:
Why move NotificationChannelWrapper & NotificationChannelWrapper out of their respective files, putting multiple classes in one file makes things less readable.

Couldn't you remove public keyword in those files instead?

Reply:
Just removing public would do it too, but the two are very small and specific, it feels very excessive to have dedicated files for them. Now that the manager classes are merged, they are used only in this Java class, kind of should be nested classes actually.